app/assets/javascripts/govuk/selection-buttons.js

```diff
@@ -6,8 +6,6 @@
   if (typeof GOVUK === 'undefined') { root.GOVUK = {}; }
 
   var SelectionButtons = function (elmsOrSelector, opts) {
-    var $elms;
```

Comment:
In case it isn't clear, this variable was getting written to, but nobody was reading from it.